src/main/java/seedu/moneygowhere/data/currency/CurrencyManager.java

```diff
@@ -0,0 +1,59 @@
+package seedu.moneygowhere.data.currency;
+
+import seedu.moneygowhere.common.Configurations;
+import seedu.moneygowhere.common.Messages;
+import seedu.moneygowhere.data.expense.Expense;
+import seedu.moneygowhere.exceptions.CurrencyInvalidException;
+
+import java.util.HashMap;
+
+/**
+ * Stores and manages a HashMap of currencies.
+ */
+public class CurrencyManager {
+    private HashMap<String, Float> exchangeRates;
```

Comment:
Do use BigDecimal instead of floats or doubles when representing currency amounts.